core/domain/topic_services.py

```diff
@@ -246,6 +284,36 @@ def save_new_topic(committer_id, topic):
         })])
 
 
+def update_subtopic_page(topic_id, subtopic_id, change_list):
+    """Applies a changelist and updates the subtopic page model entry in the
+    datastore.
+
+    Args:
+        topic_id: str. The topic id in which the subtopic is present.
+        subtopic_id: int. The id of the subtopic.
+        change_list: list(TopiChange). The list of changes that are to be
+            applied to the subtopic page object.
+    """
+    subtopic_page_id = topic_domain.SubtopicPage.get_subtopic_page_id(
+        topic_id, subtopic_id)
+    subtopic_page = get_subtopic_page_by_id(subtopic_page_id)
+    try:
+        for change in change_list:
+            if change.cmd == topic_domain.CMD_UPDATE_SUBTOPIC_PAGE_PROPERTY:
+                if (change.property_name ==
```

Comment:
Again an "else raise Error" is needed too. Cover every single possibility for a cmd could look like, including the possibly-invalid ones.

Reply:
For the outer loop (previous comment), I will add, but for the inner one, when the TopicChange object is created itself, an exception would be thrown if the property_name is not in the corresponding list in topic_domain. So, that would be enough, right?

Reply:
I think add it here as well. It's an error that's not meant to happen, I know, but it gives the reader confidence that every case is covered (even if they don't see the other part of the code you're referring to).

Reply:
Ok, done.